securedrop-workstation-svs-disp/debian/changelog-buster

```diff
@@ -0,0 +1,23 @@
+securedrop-workstation-svs-disp (0.1.3+buster) unstable; urgency=medium
+
+  * Provides Debian 10 (Buster) compatibility.
+
+ -- SecureDrop Team <securedrop@freedom.press>  Thu, 21 Nov 2019 14:15:47 -0400
+
+securedrop-workstation-svs-disp (0.1.2) unstable; urgency=medium
```

Comment:
do we want to include the 0.1.2 and before entries in the buster changelog since they were only released on stretch?

Reply:
Either is fine for me. We removed them in the other packages iirc.

Reply:
removed!